aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -3,83 +3,56 @@ title: Choose an ASP.NET Core UI
 author: wadepickett
 description: Learn when to use which ASP.NET Core web UI technologies. Understand the server, client and hybrid options.
 ms.author: wpickett
-ms.date: 09/12/2023
+ms.date: 10/04/2023
 uid: tutorials/choose-web-ui
 ---
 # Choose an ASP.NET Core web UI
 
 ASP.NET Core is a complete UI framework. Choose which functionalities to combine that fit the app's web UI needs.
 
-## Benefits vs. costs of server and client rendered UI
+## ASP.NET Core Blazor
```

Comment:
The updated Blazor section seems to be focused on giving an overview of what Blazor is all about, which we already have covered in the [updated Blazor Overview for .NET 8](https://learn.microsoft.com/aspnet/core/blazor/?view=aspnetcore-8.0). This is a solid start. The new content has the high-level structure we want. I like how the Razor Pages, MVC, and JavaScript sections are structured: they briefly describe what the framework is and then list the benefits for choosing that option. Can we make the Blazor section more like the other sections?

Reply:
Benefits of using Blazor:
- Reusable component model
- Efficient diff-based component rendering
- Flexibly render components from the client or server
- Build rich interactive web UI components in C#
- Render components statically from the server
- Progressively enhance server-rendered components for smoother navigation and form handling and to enable streaming rendering
- Share code for common logic on the client and server
- Interop with JavaScript
- Integrate components with existing MVC, Razor Pages, or JavaScript based apps

Reply:
Commited, thanks.